core/test/integration/ledger_api_blockchain_explorer_tests.cpp

```diff
@@ -170,4 +170,16 @@ TEST_F(LedgerApiEthereumLikeBlockchainExplorerTests, GetGasPrice) {
 TEST_F(LedgerApiEthereumLikeBlockchainExplorerTests, GetEstimatedGasLimit) {
     auto result = wait(explorer->getEstimatedGasLimit("0x57e8ba2a915285f984988282ab9346c1336a4e11"));
     EXPECT_GE(result->toUint64(), 10000);
-}
\ No newline at end of file
+}
+
+TEST_F(LedgerApiEthereumLikeBlockchainExplorerTests, PostEstimatedGasLimit) {
+  auto request = api::EthereumGasLimitRequest(
+      optional<std::string>(), optional<std::string>(), optional<std::string>(),
+      optional<std::string>("0xa9059cbb00000000000000000000000013C5d95f25688f8A"
```

Comment:
Since `optional` has a non-explicit constructor, you don’t have to use `optional<std::string>(_)` here. You can directly pass the address. :)